roles/metering/templates/settings.conf.j2

```diff
@@ -0,0 +1,20 @@
+[DEFAULT]
+log_level = DEBUG
+
+[Billing]
+Volume:
+    volume.size
+    volume.type
+
+Instance:
+    runtime
+    flavor
+
+[output]
+file = pushed_billing_data
+odoo =
+
+[odoo]
+odoo_db =
+odoo_user_name =
+odoo_api_key =
```

Comment:
Is this really the way the settings should look like? Or do the empty variables need to be filled in somehow? Most likely there should be role variables then that get referenced in the template?

Reply:
those variables only has to be filled in if you want to use odoo as billing sink. those fields are ignored as long as they are not used.

Reply:
Parameterization we can do in a follow up. This is rather MVP for now.